argus/scenarios/service_mock.py

```diff
@@ -268,3 +271,33 @@ def x509():
             TO9FwPXIRcR/XrcMDn7slaHtbILM3P4pIbaXUhvel+qLOAMp6k2iDl2Q
             -----END CERTIFICATE-----
             """.strip())
+
+
+class HTTPMetadataServiceApp(BaseServiceApp):
```

Comment:
It should be clear from its name what's the actual use case for this, since it can't be used as a full mocked HTTP server, similar to the other ones. Something like X509HTTPMetadataServiceApp?